formats/hocon/src/test/kotlin/kotlinx/serialization/hocon/HoconNamingConventionTest.kt

```diff
@@ -39,42 +39,42 @@ class HoconNamingConventionTest {
     }
 
     @Test
-    fun testDeserializeUsingSerialNameInsteadOfNamingConvention() {
```

Comment:
This test was successful only because the value in `@SerialName` is follow the naming convention 😄 
I think it is confusing and not helpful to check anything.